### PR TITLE
[SMALLFIX] Tachyon code update

### PR DESCRIPTION
**Pull request by caadecarvalho**
I introduced a new checking of the input parameters, so if program execute without parameters or having three or more input parameters, it will execute printUsage function and exit with error code 1. Also, I changed the IF-ELSIF-ELSE statements for a CASE which turns to be more clean and have a better readability.

### CODE REVIEW

**Comment by haoyuan** [comment]
Jenkins, test this please.

**Comment by haoyuan** [comment]
Jenkins, ok to test.

File: bin/tachyon [hunk 1]
```diff
@@ -238,18 +239,23 @@ if [[ ${COMMAND} == "format" ]]; then
...
   PARAMETER=master
-elif [[ ${COMMAND} == "formatWorker" ]]; then
+;;
+"formatWorker")
   echo "Formatting Tachyon Worker @ `hostname -f`"
   CLASS=tachyon.Format
   PARAMETER=worker
-elif [[ ${COMMAND} == "tfs" ]]; then
+;;
+"tfs")
```
Comment (yupeng9) [inline bin/tachyon]: I think tfs can take more than 2 arguments?

Comment (caadecarvalho) [inline bin/tachyon]: Totally agree with you, I was just analyzing TfsShell.java when I realized this, let's just forget about it. Also, it won't give us the opportunity to change MkdirShell.java to have two inputs. Sorry for it.

Comment (yupeng9) [inline bin/tachyon]: No worries :) The case switch change is still a good improvement to have.

**Comment by caadecarvalho** [comment]
Removed arguments limitation set on the begging of the code.

**Comment by apc999** [comment]
+1 LGTM.

Thanks for the improvement!

**Comment by yupeng9** [comment]
LGTM. Thanks for the improvement!

**Pull request merged**